ui/src/main/js/application/auth/AuthenticationConfiguration.js

```diff
@@ -351,6 +380,7 @@ const AuthenticationConfiguration = ({
                         errorValue={errors.fieldErrors[AUTHENTICATION_SAML_FIELD_KEYS.roleAttributeMapping]}
                     />
                 </CollapsiblePane>
+                {showBlackDuckSSOImportModal && importBlackDuckSSOConfigFields}
```

Comment:
This might be easier to read if you replaced the importBlackDuckSSOConfigFields with the actual HTML on the next line